test/mason/mason-test/PREDIFF

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
Assuming we use this `PREDIFF` in all mason tests, can this be a symlink to `../PREDIFF` to reduce future maintenance costs in keeping this file updated? (ditto for other uses of this `PREDIFF`)